package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "bellhop-iframe",
-  "version": "2.1.1",
-  "main": "bellhop.js",
-  "browser": "bellhop-umd.js",
+  "version": "2.1.2",
+  "main": "dist/bellhop.js",
+  "module": "dist/bellhop.js",
+  "browser": "dist/bellhop.js",
```

Comment:
Shouldn't the `main` and `browser` entries point to the UMD build?

Reply:
I tried that and I was having issues when trying to import it where it was always import in the UMD version. I suspect that it's based on the project environment you try to import it into. This was my work around so it always imported the es6 version unless you explicitly imported the UMD version.